api/keystore/service_test.go

```diff
@@ -280,3 +281,51 @@ func TestServiceExportImport(t *testing.T) {
 		}
 	}
 }
+
+func TestServiceDeleteUser(t *testing.T) {
```

Comment:
We should also have a test for creating the same user again, the user's data should have been wiped.

Reply:
Added.  thanks.